### PR TITLE
Paying for College: Front-end Bug Fixes

### PR DESCRIPTION
**Pull request by mistergone**
This PR will fix a number of bugs in the front end of the Paying for College costs tool.

## Additions
- Add handling for Expenses region dropdown
- Add handling for Send Yourself the Link button

## Changes
- Fix many content errors and typos
- Fix Action Plan button handling
- Fix Impact Question handling on Save & Finish page
- Fix handling of URL querystring

## Checklist
- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [x] Future todos are captured in comments and/or tickets
- [ ] Project documentation has been updated, potentialy one or more of:
  - [This repo’s docs](https://cfpb.github.io/cfgov-refresh/) (edit the files in the `/docs` folder) – for basic, close-to-the-code documentation on working with this repo

### Front-end testing
_[When new or significantly modified front-end functionality is present, the following things should be tested. Feel free to delete this section if not applicable to this PR.]_

### CODE REVIEW

File: cfgov/unprocessed/apps/paying-for-college/js/models/expenses-model.js [hunk 1]
```diff
@@ -44,13 +44,17 @@ const expensesModel = {
...
    */
-  setValue: ( name, value ) => {
+  setValue: ( name, value, updateView ) => {
     expensesModel.values[name] = stringToNum( value );
     expensesModel.calculateTotals();
-    updateExpensesView();
-    updateCostOfBorrowingChart();
-    updateAffordingChart();
+
+    if ( updateView === false ) {
```
Comment (virginiacc) [inline cfgov/unprocessed/apps/paying-for-college/js/models/expenses-model.js:53]: Should this be `updateView !== false`?

Comment (virginiacc) [inline cfgov/unprocessed/apps/paying-for-college/js/models/expenses-model.js:53]: Should this be `updateView !== false`?

**Comment by anselmbradford** [comment]
Run `gulp lint --fix` please.

File: cfgov/unprocessed/apps/paying-for-college/js/dispatchers/update-state.js [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-import { updateCostOfBorrowingChart, updateMakePlanChart, updateMaxDebtChart } from '../dispatchers/update-view.js';
 
 const updateState = {
 
   /**
-   * activeSection - Change the active navigation view, trigger an update
-   * for the navigationView
-   * @param {string} item - Value of 'data-nav_item' attribute
+   * activeSection - Update the app's active section
+   * @param {*} value - the value to be assigned
```
[the remark this answers is not in the document]
Comment (mistergone) [reply cfgov/unprocessed/apps/paying-for-college/js/dispatchers/update-state.js:12]: There are both Strings and Booleans being sent to the stateModel with this function. :)

File: cfgov/unprocessed/apps/paying-for-college/js/models/expenses-model.js (hunk 1 above)
Comment (mistergone) [reply cfgov/unprocessed/apps/paying-for-college/js/models/expenses-model.js:53]: Yes, you are correct! Fixing and pushing. :)

File: cfgov/unprocessed/apps/paying-for-college/js/dispatchers/update-state.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/unprocessed/apps/paying-for-college/js/dispatchers/update-state.js:12]: Trivial, but if you want to fix it, this is saying any type could be passed in (string, boolean, object, number, etc.). But if it is only strings or booleans it'd be `@param {string|boolean} value`. Uppercase those if instances are passed in (e.g. `new String()`), but it's likely primitives (`"string"`), so they'd be lowercase.

**Pull request merged**